frontend/src/components/templates/InstructorEnrollmentList.js

```diff
@@ -146,6 +147,29 @@ class InstructorEnrollmentList extends React.Component {
         return <span style={{ color }}>{daysFromToday}j</span>;
       },
     },
+    {
+      Header: <MailIcon color={'var(--datapass-dark-grey)'} />,
+      accessor: 'count_notify_events_from_demandeurs',
+      headerStyle: {
+        ...enrollmentListStyle.header,
+        ...enrollmentListStyle.updateAtHeader,
+      },
+      style: {
+        ...enrollmentListStyle.cell,
+        ...enrollmentListStyle.centeredCell,
```

Comment:
you should remove ellipsis for this collumn.
![image](https://user-images.githubusercontent.com/4190728/186438224-a5677851-f0b2-4f80-9803-8226d5d15b9b.png)


ex: 

```css
td_element {
  text-overflow: unset;
}
```